packages/video_player/video_player/README.md

```diff
@@ -136,3 +165,15 @@ and so on.
 To learn about playback speed limitations, see the [`setPlaybackSpeed` method documentation](https://pub.dev/documentation/video_player/latest/video_player/VideoPlayerController/setPlaybackSpeed.html).
 
 Furthermore, see the example app for an example playback speed implementation.
+
+### Closed Captioning
+
+#### Caption Files
+
+You can set the closed captioning on your `_controller` by calling `_setClosedCaptionFile`. Supported caption file formats are the SubRip (.srt) and WebVTT (.vtt) file formats.
+
+#### Displaying Captions
+
+You can use the `ClosedCaption` widget to display captions on a video in an overlay.
+
+You can access the caption string from the `_controller` using `_controller.value.caption.text`.
```

Comment:
(Especially since you've put this in "displaying captions", but it's not actually specific to display.)